tests/test_end_to_end.py

```diff
@@ -150,7 +150,7 @@ def test_prompt_is_not_modified_for_open_source_models(self, mock_aioresponse, m
 
     def test_gemini_specific_args_are_passed_to_model(self):
         # GIVEN
-        gemini_model_name = "gemini-model-name"
+        gemini_model_name = "gemini-1.0-pro-001"
```

Comment:
Same as above, should be `1.5` instead of `1.0`

Reply:
fixed